src/components/form/Form.vue

```diff
@@ -180,6 +182,22 @@ export default {
     },
     mixins: [dataLayerMixin],
     props: {
+        /**
+         * If set to true, the form submits through the marketo library
+         * using a hidden form. If false it submits using native browser
+         * form logic.
+         */
+        isMarketo: {
```

Comment:
Can you add unit tests for this new code?

Reply:
I'm not really sure how we'd go about testing this, it changes some logic within the submit function and the html outcome in the component is the same, plus that function is currently entirely untested. It feels like that might be something that we're supposed to approach with different tooling other than Jest. Do you have any thoughts about how we might do that @GemmaLouise ?

Reply:
@mjpvs Hmm, yeah ideally we would test that the submit is going to the right URL based on the props. I thought this would be done by mocking the API call or maybe testing the correct method has been called? Maybe something in the Jest docs about testing form submissions?

Reply:
Cool, I've added tests to ensure it calls the correct submit function and that that then succeeds if the url called returns a 200 which I think covers it